new-content/learning-resources/textbook-1-3.md

```diff
@@ -1,7 +1,7 @@
 ---
 order: 100
 title: Qiskit Textbook Chapters 1 - 3
-to: https://qiskit.org/textbook/ch-states/introduction.html
+to: https://learn.qiskit.org/course/ch-states
```

Comment:
Could you remove this change? It's just to avoid conflicts because this file in going to disappear: https://github.com/Qiskit/qiskit.org/pull/1938/files

Reply:
Done 👌